.github/workflows/build_pr_documentation.yml

```diff
@@ -4,9 +4,8 @@ name: Build PR documentation
 
 on:
   workflow_dispatch:
```

Comment:
could be removed no ?
```suggestion
```

Reply:
I think it's good to be able to trigger the PR doc build on demand so I would rather keep it